zenoh/tests/matching.rs

```diff
@@ -26,200 +28,212 @@ macro_rules! ztimeout {
     };
 }
 
+#[cfg(feature = "unstable")]
+async fn create_session_pair(locator: &str) -> Result<(Session, Session)> {
+    let config1 = {
+        let mut config = zenoh::config::peer();
+        config.scouting.multicast.set_enabled(Some(false)).unwrap();
+        config
+            .listen
+            .set_endpoints(vec![locator.clone().parse().unwrap()])
+            .unwrap();
+        config
+    };
+    let config2 = zenoh::config::client([Locator::from_str(locator)?]);
+
+    let session1 = ztimeout!(zenoh::open(config1).res_async())?;
+    let session2 = ztimeout!(zenoh::open(config2).res_async())?;
+    Ok((session1, session2))
+}
+
 #[cfg(feature = "unstable")]
 #[test]
-fn zenoh_matching_status_any() {
+fn zenoh_matching_status_any() -> Result<()> {
```

Comment:
Based on my comment below on `unwrap()`, there is no need to return a `Result`.

Reply:
Done.